docs/developers.html

```diff
@@ -0,0 +1,209 @@
+<!doctype html>
+<html class="no-js" lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>Welcome | oVirt</title>
```

Comment:
This will be more for when the jekyll site gets built but we can update the title tag with a page specific title. so this could be `<title> Developers | oVirt</title>`